editors/code/src/config.ts

```diff
@@ -64,9 +64,24 @@ export class Config {
      * `platform` on GitHub releases. (It is also stored under the same name when
      * downloaded by the extension).
      */
-    private static prebuiltLangServerFileName(platform: NodeJS.Platform): null | string {
+    private static prebuiltLangServerFileName(
+        platform: NodeJS.Platform,
+        arch: string
+    ): null | string {
+        // See possible `arch` values here:
+        // https://nodejs.org/api/process.html#process_process_arch
+
         switch (platform) {
-            case "linux":  return "ra_lsp_server-linux";
+
+            case "linux": {
+                switch (arch) {
+                    case "arm":
+                    case "arm64": return null;
+
+                    default: return "ra_lsp_server-linux";
```

Comment:
I would reverse this check: return `ra_lsp_server-linux` for `x64` and `null` in the other cases.

Same for Windows (`x64`, @matklad can confirm). It's probably not realistic to offset 32-bit builds at this time.

Reply:
Okay, thank you for the hint. And what about Mac?

Reply:
Same thing. I _assume_ it returns `x64`, but I can't test.